wandb/proto/wandb_internal.proto

```diff
@@ -136,6 +136,7 @@ message ErrorInfo {
  */
 message RunExitRecord {
   int32 exit_code = 1;
+  float runtime = 2;
```

Comment:
I think this should be an integer type as that is what we do elsewhere for tracking time.

Maybe even encode the units:
runtime_sec